packages/core/src/js/replay/mobilereplay.ts

```diff
@@ -155,13 +166,31 @@ export const mobileReplayIntegration = (initOptions: MobileReplayOptions = defau
 
   const options = mergeOptions(initOptions);
 
-  async function processEvent(event: Event): Promise<Event> {
+  async function processEvent(event: Event, hint: EventHint): Promise<Event> {
     const hasException = event.exception?.values && event.exception.values.length > 0;
     if (!hasException) {
       // Event is not an error, will not capture replay
       return event;
     }
 
+    // Check if beforeErrorSampling callback filters out this error
+    if (initOptions.beforeErrorSampling) {
+      try {
+        if (initOptions.beforeErrorSampling(event, hint) === false) {
+          debug.log(
+            `[Sentry] ${MOBILE_REPLAY_INTEGRATION_NAME} skipped due to beforeErrorSampling for event ${event.event_id}.`,
```

Comment:
You did a great naming log! But I think using `skipped` may give the impression that the replay will not be sent afterward, what do you think of the following suggestion?

```suggestion
            `[Sentry] ${MOBILE_REPLAY_INTEGRATION_NAME} not sent; beforeErrorSampling conditions not met for event ${event.event_id}.`,
```

Reply:
Sounds good 👍